analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -234,10 +258,10 @@ def process(self, events):
             # very very loosely based on https://arxiv.org/abs/2006.13076
 
             # pT > 25 GeV for leptons & jets
-            selected_electrons = events.electron[events.electron.pt > 25]
-            selected_muons = events.muon[events.muon.pt > 25]
-            jet_filter = events.jet.pt * events[pt_var] > 25  # pT > 25 GeV for jets (scaled by systematic variations)
```

Comment:
Let's keep this old comment here, it will be superseded with better systematics handling in a subsequent version but otherwise I think this `events[pt_var]` approach might look quite confusing.